client/src/router.config.tsx

```diff
@@ -56,38 +48,12 @@ export const routerConfig: Route = {
       component: TaskPage
     },
     {
-      name: 'Help',
+      key: 'docs',
+      name: 'Documentation',
       icon: <QuestionCircleOutlined />,
-      path: '/help',
-      hideChildrenInMenu: true,
-      component: HelpPage,
-      children: [
-        {
-          path: '/help/basics',
-          name: 'Basics',
-          component: BasicHelpPage,
-          authority: AUTHORITIES.ROLE_TEACHER
-        },
-        {
-          path: '/help/definitions',
-          name: 'Definition Files',
-          component: DefinitionsHelpPage,
-          authority: AUTHORITIES.ROLE_TEACHER
-        },
-        {
-          path: '/help/ide',
-          name: 'Online IDE',
-          component: IdeHelpPage,
-          authority: AUTHORITIES.ROLE_TEACHER
-        }
-      ]
-    },
-    {
-      path: '/admin',
-      name: 'Administration',
-      icon: <SettingOutlined />,
-      authority: AUTHORITIES.ROLE_ADMIN,
-      component: AdminPage
+      isUrl: true,
+      target: '_blank',
```

Comment:
can we add rel="noopener noreferrer" here?

Reply:
It's possible with a custom component